resources/queries/targetedms/QCGroupingPrecursorPeptides.sql

```diff
@@ -0,0 +1,34 @@
+-- This query for Precursor Peptides displays under QC Summary menu 'Include or Exclude Peptides/Molecules'
+
+SELECT
+*,
+(CASE
+    WHEN precPep.precursorIdentifier IN (SELECT precursorIdentifier FROM targetedms.ExcludedPrecursors) THEN 'Excluded'
+    ELSE 'Included'
+END) AS markedAs
+FROM
+     (SELECT
+        (pep.PeptideModifiedSequence || ',' || cast(prec.charge AS VARCHAR) || ',' || cast(round(prec.mz, 3) AS VARCHAR)) AS precursorIdentifier, -- "key field" required for 'requireSelection' to work
+        pep.PeptideModifiedSequence AS peptideSequence,
+        -- pep.PeptideGroupId.RunId.File,
+        pep.PeptideGroupId.Label,
+        prec.charge,
+        prec.mz,
+        prec.neutralMass,
+        prec.IsotopeLabelId.Name
+        FROM
+        targetedms.Peptide pep
```

Comment:
Same thing on this join - if you just use Precursor and get to the peptide columns via the lookup syntax it'll be faster.